### PR TITLE
fix: find wars and aars file in scan-all-unmanaged mode

### PR DESCRIPTION
**Pull request by admons**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?

Discover `aar` and `war` file in the `scan-all-unmanaged` mode
Today we say that we support this in the documentation, but the code doesn't acts accordingly

### CODE REVIEW

File: lib/jar.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export function isJar(file: string): boolean {
 export function findJars(targetPath: string, recursive = false): string[] {
   const stats = fs.statSync(targetPath);
   const dir = stats.isFile() ? path.dirname(targetPath) : targetPath;
-  return glob.sync(`${dir}/${recursive ? '**/' : ''}*.jar`);
+  return glob.sync(`${dir}/${recursive ? '**/' : ''}*.@(j|w|a)ar`);
```
Comment (admons) [inline lib/jar.ts:110]: According to [globs documentation](https://www.npmjs.com/package/glob#glob-primer)

Comment (gitphill) [reply lib/jar.ts:110]: maybe change filename and function name, `archive.ts` and `findAchives` ?

Comment (gitphill) [reply lib/jar.ts:110]: does isJar need to be same as this?

```
export function isJar(file: string): boolean {
  return !!file.match(/\.(([jwa]ar)|(zip))$/);
}
```

Comment (gitphill) [reply lib/jar.ts:110]: in fact anything that mentions 'jar' in this file is better named archive?

Comment (admons) [reply lib/jar.ts]: Done
I also added zip files to the glob since it's supported for one file in the `isJar` (now `isArchive`) function

**Review by gitphill — APPROVED** [review]

**Pull request merged**